### PR TITLE
adds test for link to merchant items page and as well adds the link i…

### LINKED ISSUE #36: User Story 36, Merchant's Items index page
User Story 37, Admin can see a merchant's dashboard

As an admin user
When I visit the merchant index page ("/merchants")
And I click on a merchant's name,
Then my URI route should be ("/admin/merchants/6")
Then I see everything that merchant would see

### PR DESCRIPTION
**Pull request by JoseLopez235**
PR Contains:
- Link in the merchant items page
- All Passing Test
closes #36

### CODE REVIEW

File: spec/features/merchant/items_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    end
+    
+    it 'I see a link to view my own items when I click that link my URI route should be "/merchant/items"' do
+      visit '/merchant'
+
+      expect(page).to have_link("My Items")
+
+      click_link "My Items"
+
+      expect(current_path).to eq("/merchant/items")
```
Comment (stillsheryl) [inline spec/features/merchant/items_spec.rb:23]: Do we want to add some items to make sure they are displayed?

**Review by c-bartell — APPROVED** [review]
Solid work!

File: spec/features/merchant/items_spec.rb (hunk 1 above)
Comment (c-bartell) [reply spec/features/merchant/items_spec.rb:23]: Good point. The user story itself is pretty vague on whether or not the items actually need to be there.

Comment (stillsheryl) [reply spec/features/merchant/items_spec.rb:23]: It actually looks like we add this in Story #42, so I will merge and start working on that story!

**Pull request merged**